build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -29,6 +29,8 @@
     <PropertyGroup>
       <IsDesktopAvailable>False</IsDesktopAvailable>
       <IsDesktopAvailable Condition=" '$(OSName)' == 'win' ">True</IsDesktopAvailable>
+      <IsCross>False</IsCross>
+      <IsCross Condition=" !$(Rid.Contains('$(OSName)')) Or !$(Rid.Contains('$(Architecture)')) ">True</IsCross>
```

Comment:
What does this property do?  It doesn't appear to be read by any other part of the build.